app/views/pages/developer-guide/pos-marketplace-template/core-concepts.liquid

```diff
@@ -128,5 +128,12 @@ Consumers are located in `app/views/partials/lib/consumers`.
 Categories can be adjusted by:
 
 1. First editing `app/views/partials/lib/queries/raw_category_list.liquid` file
-2. Then adding a translation to the `app/translations/en_categies.liquid` file
+2. Then adding a translation to the `app/translations/en_categories.liquid` file
```

Comment:
Categories management is part of the Admin UI `/admin/categories` , there shouldn't be `en_categories.liquid` anymore